tests/Platforms/AbstractPlatformTestCase.php

```diff
@@ -275,6 +275,7 @@ public function testGetCustomColumnDeclarationSql(): void
     {
         self::assertEquals(
             'foo MEDIUMINT(6) UNSIGNED',
+            /** @phpstan-ignore argument.type */
```

Comment:
This test doesn't pass all needed properties to the method and is already removed in 5.0.x.